app/src/renderer/components/wallet/SendModal.vue

```diff
@@ -115,7 +127,8 @@ export default {
     address: ``,
     amount: null,
     denom: ``,
-    num
+    num,
+    memo: null
```

Comment:
See comment, imo it's more transparent if you prefill this with `Sent via Lunie`
```suggestion
    memo: `Sent via Lunie`
```